_data/glossary/maintainer.yaml

```diff
@@ -0,0 +1,8 @@
+id: maintainer
+name: Maintainer
+tags:
+- community
+short-description: |
+  Works holistically across the project to maintain its health and success.
```

Comment:
This is what they do, not what they are.

Reply:
@abiogenesis-now Let's rewrite this.

Reply:
How about...

**short-description:** "A highly experienced contributor, active in multiple areas, who has cross area ownership and GitHub repo write access."

**long-description:** "They work holistically across the project to maintain its health and success and have made substantial contributions, both through code development and broader organizational efforts."

Reply:
works for me---I'm making the slight copyedit of saying "active in multiple areas [of Kubernetes]" but if you disagree, let me know